pkg/helm/data/values.yaml

```diff
@@ -80,6 +80,8 @@ Location: {{ .ContainerService.Location | quote }}
 RegistryStorageAccount: {{ .Config.RegistryStorageAccount | quote }}
 RegistryHTTPSecret: {{ Base64Encode .Config.RegistryHTTPSecret | quote }}
 AlertManagerProxySessionSecret: {{ Base64Encode .Config.AlertManagerProxySessionSecret | quote }}
+RegistryConsoleOAuthSecret: {{ Base64Encode .Config.RegistryConsoleOAuthSecret }}
```

Comment:
You need to be setting these in ` pkg/helm/chart/templates/Secret/sync.yaml`, too?

Reply:
I think so. I need to go one more round via all addon and sync process itself.